src/main/resources/application.properties

```diff
@@ -1,8 +1,8 @@
 spring.datasource.url=jdbc:mysql://localhost:3306/4156db
 spring.datasource.username=root
-spring.datasource.password=
+spring.datasource.password=dbuserdbuser
 spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
 spring.jpa.database-platform = org.hibernate.dialect.MySQL8Dialect
 spring.jpa.generate-ddl=true
-spring.jpa.hibernate.ddl-auto = create
+spring.jpa.hibernate.ddl-auto = update
```

Comment:
Let's keep spring.jpa.hibernate.ddl-auto = create for now, as we may alter the schema quite often recently